features/sctp/Makefile

```diff
@@ -1,7 +1,13 @@
 .PHONY: test, ginkgo
 
+deps:
+	cd test && \
+	go get && \
```

Comment:
better to use `go mod tidy` it will fetch needed dependencies and clean unused one